releasenotes/notes/ucc_imaginary-8d7d9c3322899a34.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Imaginary contributions to the UCC ansatz can be added along with the real contributions by 
+    setting the boolean flag 'include_imaginary' to True. A separate parameter is used for the real 
+    and the imaginary contribution coming from a single excitation 
```

Comment:
I think we can make things clearer here following the first sentence. While we know parameter here means parameter of the UCC ansatz circuit it could be talking about a method parameter here as it seems a bit vague to me. How about the second sentence being replaced by something like the following which also spells out what happens when you set it  i.e. doubles the number of parameters. as that can have implications for optimization.

`When True an extra ansatz parameter is added to each excitation that controls the imaginary contribution to its evolution. This setting then doubles the total number of ansatz parameters, as compared to the default setting of False.`

Reply:
Thanks, I updated the release note with your suggestion :+1: 

With regards to the example, this would be something for @pbark or @tanvipg to answer